invenio_communities/templates/semantic-ui/invenio_communities/details/header.html

```diff
@@ -12,15 +12,14 @@
 {%- from "invenio_communities/details/macros/access-status-label.html" import access_status_label -%}
 
 {% macro button_to_request_membership(community) %}
-  {% if permissions.can_request_membership %}
   {# TODO: Add relation_to_community for other flows #}
   <div
     id="request-membership-app"
-    data-community='{{ community | tojson }}'
+    data-community="{{ community }}"
```

Comment:
You can just pass the UI serialized community and it works (tested):

```suggestion
    data-community="{{ community_ui | tojson }}"
```